src/PIL/TiffImagePlugin.py

```diff
@@ -182,6 +182,43 @@
     (II, 1, (1,), 1, (12,), ()): ("I;16", "I;12"),
     (II, 0, (1,), 1, (16,), ()): ("I;16", "I;16"),
     (II, 1, (1,), 1, (16,), ()): ("I;16", "I;16"),
+    (II, 1, (1, 1), 1, (16, 16), (0,)): ("MB", "MB"),
+    (
```

Comment:
It's forced by `make lint`.

Reply:
Well then, `make lint` is wrong. ;>

This may just be my long term fight against minless conformity, but sometimes black's formatting obscures the code. And this is one of the cases.

Reply:
I would suggest just `# noqa` around whatever you don't want formatted for now … also the kids are using `ruff` these days 😄

Reply:
We can use `# fmt: off`+`# fmt: on` or `# fmt: skip` here:

https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#ignoring-sections